cpp/elasticsearch/extract_existing_fulltext_ids.sh

```diff
@@ -31,6 +31,18 @@ function GetDownloadStats() {
     echo -n $(echo "$response" | wc -c) >&2
     echo " bytes" >&2
     echo "------------------------------------" >&2
+    error=$(CheckError $response)
+    if [[ $(echo -n "$error" | wc --chars) -ne 0 ]]; then
```

Comment:
Should we also send an error email?

Reply:
The script is normally called from within a cronjob that should fail if a subscript fails.